### DISCUSSION TITLE
Category management suggestion

### DISCUSSION THREAD

**Discussion by davidovic999** [mikeberger/borg_calendar#208] [Ideas]
Hi Mike,
I'd like to make this suggestion about categories.
To manage categories more easily, it's best to create a simple interface:
At the top of the navigation panel: "Categories".
The drop-down menu indicates: "Category list".

By clicking on "Category list", a **SINGLE window** is displayed, containing :
		
1/ List of created categories

2/ and includes the following options:
- "Add" : add a new category
- "Delete" : select the category to be deleted
- "Edit" : change the category name
- "Save" : save category settings, and retain the saved category list on restart.

Thank you for your time,

**Comment by mikeberger** [comment]
The category chooser UI was written 20 years ago when I did not really know how to program a UI as was teaching myself. It is lousy.

Are you really planning to use categories? If so, wouldn't you want it to imitate what is done in some other calendar?

If I remember correctly, the feature that people really wanted back then was pretty much how google calendar supports multiple calendars on the same UI.

If I decided to enhance the category feature it would be to add multiple calendars and sync them with google calendar. but that is unlikely.

**Comment by davidovic999** [comment]
Ok, Mike. Keep going!
